code/web/src/setup/routes/user.js

```diff
@@ -26,5 +27,11 @@ export default {
     path: '/user/subscriptions',
     component: Subscriptions,
     auth: true
+  },
+
+  survey: {
+    path: '/user/survey',
+    component: Survey,
+    auth: true
```

Comment:
So this is a simple little block of code and it looks fine, but I just want to confirm: if we're using a new path for the survey is it still appearing in a modal? Or are we now re-routing the user to a new page?

Reply:
I think it does reroute to a new page. In which case, we should NOT create this extra route.

I was trying to add this so we could view the survey while we were building it, but I don't think we need to.

We can reject and close this PR.

Reply:
We can always add the route on top of `/crates` right? So `/crates` might let you pick from several crates to subscribe to, and then when you add a subscription you're routed to `/crates/survey`? Or maybe `/crates/${crate-id}-survey`? I'll close this PR if you want but just wanted to throw that out there.

Reply:
That seems like a good idea. I will make those changes.